internal/shared/clients.go

```diff
@@ -44,20 +44,16 @@ import (
 
 // ClientFactory are shared clients and configurations for use across the CLI commands (cmd) and handlers (pkg).
 type ClientFactory struct {
-	// Internal dependencies
-
-	APIInterface func() api.APIInterface
-
-	AppClient    func() *app.Client
-	Config       *config.Config
-	SDKConfig    hooks.SDKCLIConfig
-	HookExecutor hooks.HookExecutor
-	IO           iostreams.IOStreamer
-	EventTracker tracking.TrackingManager
-
-	Runtime       runtime.Runtime
-	CLIVersion    string
+	API           func() api.APIInterface
+	AppClient     func() *app.Client
 	AuthInterface func() auth.AuthInterface
+	CLIVersion    string
+	Config        *config.Config
+	EventTracker  tracking.TrackingManager
+	HookExecutor  hooks.HookExecutor
+	IO            iostreams.IOStreamer
+	Runtime       runtime.Runtime
+	SDKConfig     hooks.SDKCLIConfig
```

Comment:
There are 3 changes here:

1. Renamed `APIInterface → API`
2. Alphabetically ordered these fields
3. Removed newline breaks and unnecessary comments so indentation aligns

Reply:
Meant to send this with the review prior, but *thank you so very much*! This is a great PR to make such a change 😌 🎉